internal/riverinternaltest/riverinternaltest.go

```diff
@@ -58,7 +62,20 @@ func BaseServiceArchetype(tb testing.TB) *baseservice.Archetype {
 
 	return &baseservice.Archetype{
 		Logger:     Logger(tb),
-		Rand:       randutil.NewCryptoSeededConcurrentSafeRand(),
+		Rand:       rand,
+		TimeNowUTC: func() time.Time { return time.Now().UTC() },
+	}
+}
+
+// BaseServiceArchetypeDebug returns a new base service suitable for use in
+// tests with debug logging enabled. Useful for swapping into tests for
+// debugging, where additional context is often useful.
+func BaseServiceArchetypeDebug(tb testing.TB) *baseservice.Archetype {
```

Comment:
This is kind of nice for dropping into test cases you're having trouble with for extra logging.